logstash-core-event/spec/logstash/event_spec.rb

```diff
@@ -340,10 +340,15 @@
       end
 
       it "should warn for invalid value" do
-        expect(LogStash::Event::LOGGER).to receive(:warn).twice
+        LogStash::Event.logger = logger_mock
```

Comment:
Feels like this should be done in a `before` and the resetting `...logger = nil` should be in an `after` ?